docs/tools/flamegraphs.md

```diff
@@ -0,0 +1,167 @@
+# Generating Flamegraphs for a Running Node Service on Linux
+
+This guide assumes a running instance of Lodestar and will walk through how to generate a flamegraph for the process while running on Linux.
+
+## Modifying Lodestar
+
+Use the following two commands to install `perf` for generating the stack traces.  You may get a warning about needing to restart the VM due to kernel updates. This is nothing to be concerned with and if so, cancel out of the restart dialog.
+
+```bash
+sudo apt-get install linux-tools-common linux-tools-generic
+sudo apt-get install linux-tools-`uname -r`  # empirically this throws if run on the same line above
+```
+
+SSH into the Lodestar instance and modify `~/beacon/beacon_run.sh` to add a necessary flag.
```

Comment:
Each lodestar instance may be running in different configurations. In this case you should specify:
- Best to run Lodestar not dockerized but as a service.
- To run the service use a script that invokes node with <these flags>

Instead of actually printing such a concrete example of `beacon_run.sh` script

Reply:
Updated.